libraries/api-extractor/config/post-copy.json

```diff
@@ -0,0 +1,12 @@
+{
+  "copyTo": {
+    "./samples/api-documenter": [ 
+      "../api-documenter/gulpfile.js",
+      "../api-documenter/package.json",
+      "../api-documenter/run.cmd",
+      "../api-documenter/tsconfig.json",
+    ],
+    "./samples/api-documenter/src": [ "../api-documenter/src/**" ]
```

Comment:
See my other comment.  The idea is that api-documenter is part of api-extractor, and gets published with it, but we need Rush to see it as its own top-level project.

---
In reply to: [134040834](https://github.com/Microsoft/web-build-tools/pull/325#discussion_r134040834) [](ancestors = 134040834)